### PR TITLE
TruHealth - Certified Mental Health and Fitness Platform

### PR DESCRIPTION
**Pull request by rutefig**
# TruHealth

Nowadays many people try to search some information about their health on the internet. The issue is that the information most of the times is not accurate and can lead to wrong conclusions. This project aims to build a platform where health professionals can share information about health and users can trust the information because health professionals' will need to prove they really have knowledge about the subject of what they are talking about. To create trust in the information shared, we will build a skill verification system using email certificates. The platform will also integrate a reputation system where their clients can review anonymously their service if they prove they had a consultation with the professional. This will also help users to find the best professional for their needs.

This project can leverage Aztec's unique features by demonstrating a potential real-world application of privacy-preserving technology. The platform will create a trustless environment where users can trust the information shared by health professionals even if they don't know them, and they can be confident on having a source of truth about something that is so important and cannot be trusted from any source. It also creates a comfortable environment for users to review the professionals' service without the fear of being identified.

For this challenge we plan to implement:

- Skill verification system using email certificates for health and fitness professionals
- Professionals can share public information based on their certified skills and reputation and only reveal what they want to (name or anonymous, where studied or not)

Some of the features planned to implement for the long term:

- Reputation System for these professionals based on feedback emails, reviews, social influence
- Users can mint a subscription from a professional and professionals can tier their members according to what they paid for the subscription

### CODE REVIEW

File: TruHealth/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+This project can leverage Aztec's unique features by demonstrating a potential real-world application of privacy-preserving technology. The platform will create a trustless environment where users can trust the information shared by health professionals even if they don't know them, and they can be confident on having a source of truth about something that is so important and cannot be trusted from any source. It also creates a comfortable environment for users to review the professionals' service without the fear of being identified.
+
+For this challenge we plan to implement:
+
+- Skill verification system using email certificates for health and fitness professionals
```
Comment (jzaki) [inline TruHealth/README.md:9]: Would this be proof of having an email from a domain, or something else?

File: TruHealth/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Technical Approach
+
+- High level outline of main components
+- Any specific Aztec tools or features you used
+
+## Expected Outcomes
+
+What do you hope to achieve by the end of the challenge(s)?
+
+Having a MVP with the first planned features and being able to keep working on the project and scale it to a real-world application, making users use blockchain and zero knowledge without noticing it.
```
Comment (jzaki) [inline TruHealth/README.md:41]: Great point here. Definitely valuable to surface the capability benefits, without needing the user to do extra work due to the technology.

File: TruHealth/README.md (hunk 1 above)
Comment (rutefig) [reply TruHealth/README.md:9]: Here I think it needs more than domain check, some content that actually shows the professional certification

**Comment by rutefig** [comment]
Hey sorry I'm just gonna close the PR since I didn't have the time to implement it. I still hope to build this anyways with Aztec later

And thank you for the feedback @jzaki :)

**Comment by jzaki** [comment]
@rutefig Thanks for looping back on this. The discord is there if you need it, and feel free to tag me or dm when you're back on this.

**Pull request closed without merging**